src/corelocationui.cs

```diff
@@ -0,0 +1,49 @@
+using CoreGraphics;
+using ObjCRuntime;
+using Foundation;
+using UIKit;
+
+using System;
+
+namespace CoreLocationUI {
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum CLLocationButtonIcon : long {
+		None = 0,
+		ArrowFilled,
+		ArrowOutline,
+	}
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum CLLocationButtonLabel : long {
+		None = 0,
+		CurrentLocation,
+		SendCurrentLocation,
+		SendMyCurrentLocation,
+		ShareCurrentLocation,
+		ShareMyCurrentLocation,
+	}
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
```

Comment:
`API_AVAILABLE(ios(15.0),watchos(8.0))`
I guess it's not in the watchOS header ?